docs/dyn-services.md

```diff
@@ -0,0 +1,10 @@
+# Dynamic services
+
```

Comment:
Maybe define what a legacy and modern service is then.
such as:
- legacy dynamic service:
  - is managed by the director-v0
  - can be 1 or more docker services that can run anywhere in the cluster
- modern dynamic service:
  - the service is managed via the dynamic-sidecar by the director-v2
  - is composed of at least a dynamic-sidecar that act as a pod controller
  - is composed of at least a reverse-proxy that act as the service web entrypoint
  - can be 1 or more docker containers that run on the same node as the dynamic-sidecar